ci/infra/testrunner/tests/test_deployments.py

```diff
@@ -1,21 +1,26 @@
-from skuba import Skuba
-from kubectl import Kubectl
 import pytest
-import time
-from timeout_decorator import timeout
+import json
+import requests
+
+from kubectl import Kubectl
+
+
+def test_nginx_deployment(deployment, kubectl):
+    workers = 3 
```

Comment:
Why hard code this value? you can retrieve it from skuba (I think you were doing this in a previous version)
```
def test_nginx_deployment(deployment, skuba, kubectl): 
      workers = skuba.num_of_nodes("worker")
```